tests/test_local_settings.py

```diff
@@ -42,6 +43,17 @@ def pod_mutation_hook(pod):
     pod.namespace = 'airflow-tests'
 """
 
+SETTINGS_FILE_MUST_HAVE_OWNER_POLICY = """
+def task_must_have_owners(task: BaseOperator):
+    from airflow.configuration import conf
+    if not task.owner or \
+      task.owner.lower() == conf.get('operators', 'default_owner'):
+        raise AirflowClusterPolicyViolation(
+            f'''Task must have non-None non-default owner.
+            Current value: {task.owner}'''
```

Comment:
```suggestion
            f'Task must have non-None non-default owner. Current value: {task.owner}'
```

Can be just a single line since we have 110 line as max-length